patches/core/ungoogled-chromium/block-trk-and-subdomains.patch

```diff
@@ -45,20 +45,25 @@
        (static_cast<int>(text.length()) > after_scheme_and_colon)) {
      // Obtain the URL prefixed by view-source or blob and parse it.
      base::string16 real_url(text.substr(after_scheme_and_colon));
-@@ -599,7 +600,9 @@ int AutocompleteInput::NumNonHostCompone
+@@ -623,11 +624,15 @@ int AutocompleteInput::NumNonHostCompone
+ 
+ // static
  bool AutocompleteInput::HasHTTPScheme(const base::string16& input) {
-   std::string utf8_input(base::UTF16ToUTF8(input));
-   url::Component scheme;
--  if (url::FindAndCompareScheme(utf8_input, kViewSourceScheme, &scheme)) {
-+  if (url::FindAndCompareScheme(utf8_input, url::kTraceScheme, &scheme)) {
++  if (HasScheme(input, url::kTraceScheme))
 +    return false;
-+  } else if (url::FindAndCompareScheme(utf8_input, kViewSourceScheme, &scheme)) {
-     utf8_input.erase(0, scheme.end() + 1);
-   }
-   return url::FindAndCompareScheme(utf8_input, url::kHttpScheme, nullptr);
+   return HasScheme(input, url::kHttpScheme);
+ }
+ 
+ // static
+ bool AutocompleteInput::HasHTTPSScheme(const base::string16& input) {
++  if (HasScheme(input, url::kTraceScheme))
++    return false;
+   return HasScheme(input, url::kHttpsScheme);
```

Comment:
It shouldn't!  I included that since the previous iteration of the patch was doing the same check and I figured there may have been reason for it in the past.  I'll do some testing today to make sure it's safe to remove.